modules/hyprpaper/hm.nix

```diff
@@ -0,0 +1,21 @@
+{
+  config,
+  lib,
+  ...
+}: let
+  settings = {
+    preload = ["${config.stylix.image}"];
+    wallpaper = [" ,${config.stylix.image}"];
+  };
+in {
```

Comment:
Consider inlining this variable as it is used only once.

Reply:
Good idea. Done.